packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -17,7 +17,22 @@ import {
   UseAutocompleteProps,
   AutocompleteValue,
 } from "@mui/material";
-import { memo, useCallback, useMemo, useRef } from "react";
+import React, {
+  createContext,
+  FC,
+  forwardRef,
+  memo,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+} from "react";
```

Comment:
I notice we're importing `React` and also importing properties of `React`. Can we just import the properties instead of the full object? I saw `React.ReactElement` used, but we could just import `type ReactElement` instead.

Reply:
Yep, I'll get this cleaned up